jest.config.js

```diff
@@ -1,6 +1,17 @@
 module.exports = {
   rootDir: '.',
   projects: [
+    {
+      displayName: 'terra-sdk',
+      preset: 'ts-jest',
+      testEnvironment: 'node',
+      testMatch: ['<rootDir>/packages-ts/terra-sdk/**/*.test.ts'],
+      globals: {
+        'ts-jest': {
+          tsconfig: '<rootDir>/packages-ts/terra-sdk/tsconfig.json',
```

Comment:
IMO we should have one main TS (yarn) workspace for this monorepo and all! packages (something published) should be listed flat under `./packges-ts`.

This will lower the complexity of using, finding, and actually implementing packages. They are all in the same place, properly configured once.

There can be other TS projects in the monorepo outside `packages-ts` but those are not published packages (dependencies of other projects), but examples, tests & projects that are used directly.

Here is an example complex monorepo that has a similar structure:

- All TS published packages under ./packages: https://github.com/ethereum-optimism/optimism/tree/develop/packages
- For non-published packages, there could be other subfolders with TS code, for example:https://github.com/ethereum-optimism/optimism/tree/develop/integration-tests